### PR TITLE
convert java authentication guide.

### PR DESCRIPTION
**Pull request by bmarwell**

### CODE REVIEW

File: jbake/shiro-website.iml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <output url="file://$MODULE_DIR$/target/classes" />
+    <output-test url="file://$MODULE_DIR$/target/test-classes" />
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/assets/static" />
+      <excludeFolder url="file://$MODULE_DIR$/target" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (bdemers) [inline jbake/shiro-website.iml:13]: Should this file be added to `.gitignore`?

**Review by bdemers — APPROVED** [review]
+1, pending review of the intellij project file

@bmarwell adding the TOC was a great addition, I tweaked the structure slightly after seeing how it rendered in the TOC

Also, I tweaked the definition to use the "glossary" syntax `<word>:: <definition>` (if you prefer the look of the previous code let me know, and i can revert that)

File: jbake/shiro-website.iml (hunk 1 above)
Comment (bmarwell) [reply jbake/shiro-website.iml:13]: Yes!

**Comment by bmarwell** [comment]
Thanks, Brian!

**Pull request merged**